05-angularjs/scripts/app.js

```diff
@@ -0,0 +1,9 @@
+(function () {
+ 
+  var app = angular.module('movieApp',[
```

Comment:
This is not right, controller, directives and other stuff must not be a new module a module has its own directives, controllers, etc. Review the work and try to fix this :)